test/function/target.cpp

```diff
@@ -998,6 +998,30 @@ namespace {
     stumpless_free_all(  );
   }
 
+  TEST( SetOption, Cons ) {
+    struct stumpless_target *target;
+    struct stumpless_target *target_result;
+    char buffer[100];
+    int option;
+
+    target = stumpless_open_buffer_target( "test target",
+                                           buffer,
+                                           sizeof( buffer ) );
+    ASSERT_TRUE( target != NULL );
```

Comment:
Consider changing this to `ASSERT_NOT_NULL`. This isn't completely consistently used across the test base yet, but I am pushing to move it in that direction for new tests.

Reply:
While I am at it I'll make this change for the rest of the cases in `test/function/target.cpp` which are using `ASSERT_TRUE` to check for `NULL`.